.travis.yml

```diff
@@ -1,6 +1,7 @@
 language: node_js
 node_js:
   - '0.10'
+  - '0.12'
```

Comment:
@phated Ok, I hope I'll have some time later this week to migrate some repos to eslint and update their `.travis.yml` while I'm at it. Is [this](https://github.com/phated/undertaker/blob/master/.eslintrc) the `.eslintrc` you want throughout?

For what it's worth, my fork does trigger a travis rebuild on commits, [link](https://travis-ci.org/erikkemperman/async-settle). So the config is fine. Maybe you want to take a look at the settings on travis-ci.org for the main repo, I suspect that something needs doing there? Even just removing and re-adding the repo might help.